src/js/flows/viewLogDetail.ts

```diff
@@ -1,16 +1,27 @@
-import {uidSearch} from "./searches/uidSearch"
 import LogDetails from "../state/LogDetails"
 import {Thunk} from "../state/types"
 import {zng} from "zealot"
 import {isEqual} from "lodash"
+import {fetchCorrelation} from "../correlation/flows/fetch"
+import Notice from "../state/Notice"
+import ErrorFactory from "../models/ErrorFactory"
+import Current from "../state/Current"
 
-export const viewLogDetail = (log: zng.Record): Thunk => (
+export const viewLogDetail = (record: zng.Record): Thunk => (
   dispatch,
   getState
 ) => {
   const current = LogDetails.build(getState())
-  if (!isEqual(log, current)) {
-    dispatch(uidSearch(log))
-    dispatch(LogDetails.push(log))
+  if (!isEqual(record, current)) {
+    dispatch(LogDetails.push(record))
+
+    if (!Current.getSpaceId(getState())) return // It is possible for this code to be called without a space
+    dispatch(LogDetails.updateUidStatus("FETCHING"))
+    dispatch(fetchCorrelation(record))
+      .then((records) => {
+        dispatch(LogDetails.updateUidLogs(records))
+        dispatch(LogDetails.updateUidStatus("SUCCESS"))
```

Comment:
The name "uidLogs" is wrong now since it could be uid or community_id logs. I think "relatedLogs" would be better. A future PR...